code/datums/mutations/body.dm

```diff
@@ -224,17 +224,35 @@
 
 /datum/mutation/human/strong
 	name = "Strength"
-	desc = "The user's muscles slightly expand."
+	desc = "The user's muscles slightly expand, allowing them to move heavy objects easily."
 	quality = POSITIVE
-	text_gain_indication = "<span class='notice'>You feel strong.</span>"
+	text_gain_indication = "<span class='notice'>You feel strong!</span>"
 	difficulty = 16
+	instability = 20
+
+/datum/mutation/human/strong/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	ADD_TRAIT(owner, TRAIT_NOT_SLOWEDBYDRAG, "genetics") // Don't get slowed by dragging lockers or mobs
```

Comment:
```suggestion
	owner.add_movespeed_mod_immunities(type, /datum/movespeed_modifier/bulky_drag)
	owner.add_movespeed_mod_immunities(type, /datum/movespeed_modifier/human_carry)
```

Reply:
ah, thanks, that's what i was looking for when i was trying to add carry slowdown immunity